src/plugins/openmeteo/register.py

```
@@ -0,0 +1,5 @@
+from .datasource import OpenMeteoDataSource
+
+
+def register_plugin(registry):
```

Ruff (0.12.2)

Missing return type annotation for public function `register_plugin`

Add return type annotation: `None`

(ANN201)